### DISCUSSION TITLE
Buttons

### DISCUSSION THREAD

**Discussion by kyronjavier** [taydz/Tafe230213#5] [General]
Would you be alright with all our calculators having the same looking buttons as the main menu?

**Comment by taydz** [comment]
yes, I think we should keep the same buttons throughout the program

**Comment by kyronjavier** [comment]
All right, I was also thinking of showing the calculation history upon click event. What do you think?

**Reply by taydz** [reply]
that sounds like a great idea. I'd be happy for you at add that in.
